rasa/nlu/featurizers/sparse_featurizer/count_vectors_featurizer.py

```diff
@@ -674,6 +857,8 @@ def load(
         **kwargs: Any,
     ) -> "CountVectorsFeaturizer":
 
+        finetune_mode = kwargs.pop("should_finetune", False)
```

Comment:
let's specify the param directly in the method parameters so you don't need to access the `kwargs`